documentation/staging/content/_index.md

```diff
@@ -51,28 +51,8 @@ by the operator Helm chart, Kubernetes resources created by the operator, and th
 maintain compatibility for three releases, except in the case of a clearly communicated deprecated feature, which will be
 maintained for one release after a replacement is available.
 
-### About this documentation
+#### Getting help
 
-This documentation includes sections targeted to different audiences.  To help you find what you are looking for more easily,
-please consult this table of contents:
+The WebLogic Kubernetes Operator is open source, licensed under the Universal Permissive License (UPL), Version 1.0. WebLogic Kubernetes Operator support is available via community support and via Oracle Support for Oracle WebLogic Server. To access Oracle support for running WebLogic Server domains on Kubernetes platforms, see [WebLogic Server Certifications on Kubernetes in My Oracle Support Doc ID 2349228.1](https://support.oracle.com/epmos/faces/DocumentDisplay?_afrLoop=208317433106215&id=2349228.1&_afrWindowMode=0&_adf.ctrl-state=c2nhai8p3_4).
```

Comment:
The "open source" and UPL comment seems out of context here, and the MOS note is already mentioned in the `get-help.md`.  Can we move we move the open-source/UPL comment into the introduction and remove the MOS note reference?

Reply:
For example, add a 
```
### Open source statement
The WebLogic Kubernetes Operator is open source, licensed under the Universal Permissive License (UPL), Version 1.0. 
```

Reply:
Spoke to Tom and worked out an agreement.